server/src/main/java/io/littlehorse/common/model/metadatacommand/subcommand/PutWfSpecRequestModel.java

```diff
@@ -61,6 +66,7 @@ public void initFrom(Message proto, ExecutionContext context) {
         PutWfSpecRequest p = (PutWfSpecRequest) proto;
         name = p.getName();
         entrypointThreadName = p.getEntrypointThreadName();
+        allowedUpdateType = p.hasAllowedUpdates() ? p.getAllowedUpdates() : AllowedUpdateType.ALL;
```

Comment:
suggestion:
what if we add a private constant for AllowedUpdateType.ALL?

maybe smth like this:

private static AllowedUpdateType defaultUpdateType = ...